tensorflow/lite/g3doc/guide/build_rpi.md

```diff
@@ -81,8 +81,8 @@ PATH=../rpi_tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/bin:$PATH \
 
 ## Compile natively on Raspberry Pi
 
-The following instructions have been tested on Raspberry Pi Zero, Raspbian
-GNU/Linux 10 (buster), gcc version 8.3.0 (Raspbian 8.3.0-6+rpi1):
+The following instructions have been tested on Raspberry Pi Zero, Raspberry Pi OS (previously called Raspbian)
+GNU/Linux 10 (Buster), gcc version 8.3.0 (Raspbian 8.3.0-6+rpi1):
```

Comment:
"Raspbian 8.3.0-6+rpi1" is this still valid?

Reply:
Good question. To check whether this version string is still valid we'll either need to run `gcc --version` on a fresh install of Raspberry Pi OS 10 or find some documentation online. I don't have a spare micro SD card to test this with, so I don't have an immediate answer.